src/main/java/org/terasology/signalling/components/SignalTimeDelayModifiedComponent.java

```diff
@@ -19,7 +19,11 @@
 import org.terasology.world.block.ForceBlockActive;
 
 /**
+ *
+ * Responsible for keeping track of whether a TimeDelay in a Signal has been modified.
  * @author Marcin Sciesinski <marcins78@gmail.com>
```

Comment:
As a minor side-note feel free to remove `@author` tags anywhere you find them. We made a convention to not use those some time ago. Not related to the task but if you're editing a file anyway, hey :-)